lstchain/io/io.py

```diff
@@ -750,7 +750,8 @@ def write_dataframe(dataframe, outfile, table_path, mode="a", index=False, confi
     meta: global metadata
     """
 
-    filters = tables.Filters(complevel=complevel, complib=complib)
+    filters = HDF5_ZSTD_FILTERS.copy()
```

Comment:
`HDF5_ZSTD_FILTERS` still sets level 5 by default. Should the value be updated there as well?

Reply:
That was probably a good choice for other data, no?
Wasn't this tested at the time?

Reply:
you're right. Then since this complevel is a new argument, it has to be updated in all its usages. Otherwise, it will be always level 1 by default

Reply:
Could you clarify?

Reply:
Sorry, my bad. `write_dataframe` is only used to write DL2 files as far as I can tell from the usage of the function. `complevel` defaults to 1 in this function. It reads the `HDF5_ZSTD_FILTERS` where `complevel` is defined as 5.
None of the usages of `write_dataframe` set `complevel` differently than 1. So IMO, leaving the 5 by default there, where right now is not being used at all, is going to be confusing.